### PR TITLE
doc: add minutes for meeting 31 Jan 2018

### PR DESCRIPTION
**Pull request by mhdawson**
First copy/paste of minutes, may need some fixup.

### CODE REVIEW

File: meetings/2018-01-31.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+* Rich Trott @Trott (TSC)
+* Sakthipriyan Vairamani @thefourtheye (TSC)
+* Trevor Norris @trevnorris (TSC)
+* Matteo Collina @mcollina (TSC)
+
+## Agenda
+
+### Announcements
+ 
+*Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
```
Comment (Trott) [inline meetings/2018-01-31.md]: I'll update it in this PR, but can you update the template to use `labeled` (preferred US English spelling)?

**Comment by Trott** [comment]
@mhdawson Two things:

* Can you update the template to use `labeled` (preferred US English spelling) instead of `labelled` in two places? (I've updated the PR here already.)

* We are still seeing things like `don.t` instead of `don't` when you take minutes. Not sure what that's about, but FYI and if there's anything you can do about it, cool. (I'll fix them in this PR right now.)

**Comment by mhdawson** [comment]
@Trott the ' -> . seenms to happen because of my cut and paste from chrome to a putty session.  Not sure why it happens. I try to  find/fix them but obviously don't catch them all.

I  can only find one instance of 'labeled', which I've updated in the template.

File: meetings/2018-01-31.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+* **Recording**: https://www.youtube.com/watch?v=a6YtLIaqMqs 
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/478
+* **Minutes Google Doc**: https://docs.google.com/document/d/1R7Z9Rt01fqFddMnllJddezKwGIbaCcrkrGfw5Lhc_1A
+
+## Present
+
+* Ali Ijaz Sheikh @ofrobots (TSC)
+* Daniel Bevenius @danbev (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* ......... ...... ......... @ChALkeR (TSC)
```
Comment (TimothyGu) [inline meetings/2018-01-31.md]: Hmm

Comment (mhdawson) [reply meetings/2018-01-31.md]: I thought I had fixed that, but seems not to have worked.

Comment (mhdawson) [reply meetings/2018-01-31.md]: Looks ok if I use "view".

Comment (mhdawson) [reply meetings/2018-01-31.md]: So I think its ok.

Comment (Trott) [reply meetings/2018-01-31.md]: I fixed it after Timothy's comment.

**Comment by mhdawson** [comment]
Landed as a886aa1c7987518203f0d1b7360b9a305c602894

File: meetings/2018-01-31.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2018-01-31
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=a6YtLIaqMqs
```
Comment (reedloden) [inline meetings/2018-01-31.md]: This should be https://www.youtube.com/watch?v=bIzYZhXRCa0

**Pull request closed without merging**